atk/Gir.toml

```diff
@@ -67,6 +66,13 @@ name = "Atk.Action"
 status = "generate"
 trait_name = "AtkActionExt" # duplicate with gio
 
+[[object]]
+name = "Atk.Document"
+status = "generate"
+    [[object.function]]
+    name = "get_attributes"
+    ignore = true # TODO: re-enable once AttributeSet is properly handled
```

Comment:
Why do you ignore them all if they can be implemented? Just leave them around, then at least we are aware that they still need to be bound if someone cares enough (and they should up in the "unbound" mode of gir)

Reply:
Because of the docs, it will create broken links otherwise

Reply:
Annoying

Reply:
Ideally gir would mark those functions/types as not possible to bound so we can ignore them when generating docs links

Reply:
Currently they keep their generate state